### PR TITLE
the draft pr to demonstrate the performance of `polars` lazyframes on 20240516

### PR DESCRIPTION
**Pull request by maxim-uvarov**
this pr is not intended to be merged.

### CODE REVIEW

File: book/dataframes.md [hunk 1]
```diff
@@ -88,58 +87,57 @@ We can have a look at the first lines of the file using [`first`](/commands/docs
...
-```nu no-run
-> timeit {polars open .\Data7602DescendingYearOrder.csv}
-601ms 700us 700ns
+```nu
+> timeit {polars open Data7602DescendingYearOrder.csv | polars collect; null}
+11sec 994ms 700µs 125ns
 ```
 
-This time it took us 0.6 seconds. Not bad at all.
+This time it took us 0.00053 seconds. Not bad at all.
```
Comment (fdncred) [inline book/dataframes.md:140]: where did this time come from? above it says 11 sec, which I don't understand either. I'm not sure how dataframes would be slower than nushell opening the same file.

Comment (maxim-uvarov) [reply book/dataframes.md:140]: 11 seconds is the execution time inside numd script. I executed it manually now and it shows 5 seconds which is also above expectations. 

0.00053 seconds is the time it took to create a lazyframe plan. I will fix it in the next commit.

**Comment by fdncred** [comment]
This is my timings from my Windows desktop that is not super fast. I didn't do the python timings.

```nushell
❯ timeit {open Data7602DescendingYearOrder.csv}
5sec 244ms 662µs 400ns

❯ timeit {polars open Data7602DescendingYearOrder.csv | polars collect; null}
239ms 526µs 800ns

❯ timeit {
    open 'Data7602DescendingYearOrder.csv'
    | group-by year --to-table
    | update items {|i|
        $i.items.geo_count
        | math sum
    }
}
10sec 906ms 680µs 300ns

❯ cat load.nu
let df = polars open Data7602DescendingYearOrder.csv
let res = $df | polars group-by year | polars agg (polars col geo_count | polars sum)
$res | polars collect
❯ timeit {source load.nu}
155ms 678µs 500ns
```

**Comment by fdncred** [comment]
Just FYI - Regular nushell tables can emulate the polars output too.
nushell
image
polars plugin (had to add sort to get them in the same order)
image

**Pull request closed without merging**